assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -30,41 +46,143 @@ const Title = styled.h3`
 	font-size: 19px;
 	line-height: 20px;
 	line-height: 1.4em;
-	flex: 2 0 0;
+	flex: 3 0 0;
 `;
 
 const Header = styled.div`
 	display: flex;
-	margin: 0px 4px 10px;
+	margin: 0px 0px 25px;
 `;
 
 const Message = styled.div`
-	 color: ${ ( { theme } ) => theme.colors.fg.v1 };
+	color: ${ ( { theme } ) => theme.colors.fg.v1 };
 	font-size: 19px;
 `;
 
+const FilterButtons = styled.div`
+	border-bottom:  2px solid ${ ( { theme } ) => theme.colors.mg.v1 };
+	padding: 18px 0px;
+	margin: 10px 0px 15px;
+`;
+
+const FilterButton = styled.button`
+	border: 0px;
+	background: none;
+	padding: 0px;
+	margin: 0px 28px 0px 0px;
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	font-size: 13px;
+	${ ( { active } ) => active && ` color: #fff; font-weight: bold; ` };
```

Comment:
Should the color here also come from the theme colors?

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3907/commits/d02b0b72525a12bbc64d5ee2412ec3ea958a0665